shared/teams/channel/index.tsx

```diff
@@ -51,6 +52,7 @@ const Channel = (props: Props & TabProps) => {
 
   return (
     <Kb.Box style={styles.container}>
+      {Styles.isMobile && <MobileHeader />}
       <Kb.SectionList
```

Comment:
this is covering the back button, it needs an added `top: 40`

Reply:
If this wants to work with the selection popup it should be `marginTop: 40`

Reply:
Upon posting that I realize that I may have broken clicking on the back button in my other PR, so perhaps ignore my above comment :/